src/crs_linter/cli.py

```diff
@@ -313,6 +329,34 @@ def main():
     parsed = read_files(files)
     txvars = {}
 
+    if args.tests is not None:
+        # read existing tests
+        testlist = glob.glob(args.tests)
+        testlist.sort()
+        if len(testlist) == 0:
+            logger.error(f"Can't open files in given path ({args.tests})!")
+            sys.exit(1)
+        # read the exclusion list
+        test_exclusion_list = []
+        try:
+            with open(args.filename_tests_exclusions, "r") as f:
+                for line in f:
+                    line = line.strip()
+                    if line and not line.startswith("#"):
+                        test_exclusion_list.append(line)
+        except FileNotFoundError:
+            logger.error(f"Exclusion list file '{args.filename_tests_exclusions}' not found. Skipping...")
+            sys.exit(1)
+        test_cases = {}
+        # find the yaml files
+        # collect them in a dictionary and check for test
+        for t in testlist:
+            testlist = glob.glob(t + "/*.yaml")
+            testlist.sort()
+            for tc in testlist:
+                tcname = tc.split("/")[-1].split(".")[0]
+                test_cases[int(tcname)] = 1
```

Comment:
Don't assume test file names to be rule numbers. We do that in CRS (for now), but it's a fragile convention. Either treat the filename as a string or parse the YAML and read the `rule_id` field from each YAML object.